src/TicketingSystem.BusinessLogic/Services/IPaymentService.cs

```diff
@@ -10,16 +10,39 @@ namespace TicketingSystem.BusinessLogic.Services
 {
     public interface IPaymentService : IService<Payment, PaymentDto>
     {
-        public Task<List<EventSectionSeatsModel>> GetPaymentEventSeats(string paymentId, CancellationToken cancellationToken = default);
+        public List<EventSectionSeatsModel> GetPaymentEventSeats(PaymentDto payment, CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Deletes a seat from Cart or throws the exception if there are no seats in the cart.
+        /// </summary>
+        /// <param name="seatId"></param>
+        /// <param name="cartId"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
```

Comment:
It is better to either fill XML comments if you add them, or remove at all.

Reply:
I added them in IPaymentService to make some hints on hover to see the purpose of the method

Cleaned empty params tags